backend/src/main.py

```
@@ -81,12 +72,89 @@ def internal_error(error):
+    auto_trader_service.set_loop(background_loop)
+
+    # Schedule background tasks
+    # Access properties to initialize them within the loop context
+    _ = data_fetcher_service.http_client
+    _ = ai_analysis_service.http_client
+    _ = trading_service.solana_client
+    _ = trading_service.http_client
+    _ = wallet_service.solana_client
+    _ = wallet_service.http_client
+
+    background_loop.create_task(mempool_monitor_service.start_monitoring())
```

Ruff (0.15.12)

[warning] Store a reference to the return value of `background_loop.create_task`

(RUF006)